sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLQueryPlanner.java

```diff
@@ -65,9 +65,12 @@ public ZetaSQLQueryPlanner(FrameworkConfig config) {
     plannerImpl = new ZetaSQLPlannerImpl(config);
   }
 
-  public ZetaSQLQueryPlanner(JdbcConnection jdbcConnection, RuleSet[] ruleSets) {
+  // Called by BeamSqlEnv.instantiatePlanner() reflectively
```

Comment:
Consider making this a javadoc comment so we can add a link.

Reply:
Done.